tests/integration/docker/build.bats

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+SRC="${BATS_TEST_DIRNAME}/../../lib/"
+IMG_NAME="ccbuildtests"
+
+
+setup() {
+	source $SRC/test-common.bash
+	runtime_docker
+}
+
+teardown () {
+	echo "teardown:"
+	$DOCKER_EXE rmi $IMG_NAME
+}
+
+@test "docker build env vars" {
+	var_value="test_env_vars"
+
+	run $DOCKER_EXE build -t "${IMG_NAME}" - <<EOF
+	FROM busybox
+	ENV VAR "${var_value}"
+	RUN sh -c 'echo $VAR'
+EOF
+	echo output: "${output}"
+	echo "${output}" | grep "${var_value}"
```

Comment:
Silly question. Did we 'negative test' this? I just have a niggle that if it failed, maybe it would return something like 'could not set test_env_vars' in the output, and that would still match the grep? Maybe we can set the var to something more comprehensive (guessing: test_env_vars=test_value or similar) and/or make the grep more complex to ensure we are picking up the env setting and not an error message? Just a thought.